api/views/user_views.py

```diff
@@ -754,6 +756,37 @@ def get(self, request, recording_id):
         return Response(serializer.data, status=status.HTTP_200_OK)
 
 
+@extend_schema(tags=["Recordings"])
+class GetQuizzesAndSummaries(APIView):
+    permission_classes = [IsRecordingOwner]
+
+    @extend_schema(
+        operation_id="get-quizzes-and-summaries",
+        summary="Get quizzes and summaries in chronological order.",
+        description="Returns all quizzes and summaries associated with a specific recording ID.",
+        responses={
+            200: inline_serializer(
+                "Quizzes and Summaries",
+                {"quiz": QuizSerializer(many=True), "summary": LectureSummarySerializer(many=True)},
+            ),
```

Comment:
I noticed that the inline serializer that you provided doesn't match the format of the response (quiz vs quizzes and summary vs summaries)

It we were using another serializer for the response (that includes both QuizSerializer and LectureSummarySerializer), we wouldn't have encountered this issue.

Let's define a new serializer (instead of using the inline serializer) and use it for both Swagger and the response.